tests/test_workspace.py

```diff
@@ -0,0 +1,86 @@
+from pathlib import Path
+
+import pytest
+
+from autogpt.workspace import Workspace
+
+_WORKSPACE_ROOT = Path("home/users/monty/auto_gpt_workspace")
```

Comment:
Cross platform this up

Reply:
This will resolve to a windows path if run on windows.  The `unix-y` part is just a relative path, which is unimportant for testing as everything is nested in a tempdir which will come in as an OS appropriate path from the system root.